pkg/apis/storage/v1/util/helpers.go

```diff
@@ -51,3 +54,10 @@ func IsDefaultAnnotation(obj metav1.ObjectMeta) bool {
 
 	return false
 }
+
+// IsBindingModeWaitForFirstConsumer returns true if the VolumeBindingMode is set
+// to VolumeBindingWaitForFirstConsumer
+func IsBindingModeWaitForFirstConsumer(class *storagev1.StorageClass) bool {
```

Comment:
Under what conditions do we have a StorageClass with a nil mode?  Shouldn't that be handled in defaulting?  Ie why do we need this?

Reply:
I got the impression that it is not possible to default new alpha fields.  The process now requires that we drop all alpha fields before storing them in etcd if you don't have the feature gate enabled.